plugin/src/main/java/com/hp/octane/plugins/jetbrains/teamcity/factories/ModelCommonFactory.java

```diff
@@ -61,10 +63,13 @@ public CIJobsList createProjectList() {
 				logger.info("Fetching data for project: " + buildType.getName());
 				if (!ids.contains(buildType.getInternalId())) {
 					ids.add(buildType.getInternalId());
+					List<CIParameter> ciParameters = parametersFactory.obtainFromBuildType(buildType)
+							.stream().filter(parameter -> !parameter.getName().equals(OCTANE_AUTO_ACTION_EXECUTION_ID))
+							.collect(Collectors.toList());
```

Comment:
We have our parameter execution_id that users should not be able to override in Octane, so here we filter parameters that we want to see in Octane
![image](https://user-images.githubusercontent.com/71069933/170189114-ae7e5ed8-52a3-4b8e-bf1b-ece64435005e.png) want to see all except execution_id

Reply:
@nissimshitrit yeah, it is better to filter them on the Octane side, thank you.